lightningd/options.c

```diff
@@ -228,23 +229,26 @@ static char *opt_add_announce_addr(const char *arg, struct lightningd *ld)
 		return tal_fmt(NULL, "address '%s' is not announcable",
 			       arg);
 
-	/* gossipd will refuse to announce the second one, sure, but it's
-	 * better to check and fail now if they've explicitly asked for it. */
+	/* Limit the number of addresses we can announce via options
+	 * by 2 per type to prevent spam. */
```

Comment:
I've removed this limit altogether in a separate PR (#4902), which I think is better?

I aim to apply that today (it also removed tor v2 support), so just drop this on rebase?

Reply:
okay, will do rebase when TOR2 is removed on master.